README.md

```diff
@@ -1,20 +1,157 @@
-[![Build Status](https://img.shields.io/azure-devops/build/responsibleai/fairlearn/6/dev?failed_label=bad&passed_label=good&label=GatedCheckin%3ADev)](https://dev.azure.com/responsibleai/fairlearn/_build/latest?definitionId=1&branchName=dev) ![MIT license](https://img.shields.io/badge/License-MIT-blue.svg) ![pypi badge](https://img.shields.io/badge/pypi-0.2.0-blue)
+[![Build Status](https://dev.azure.com/responsibleai/fairlearn/_apis/build/status/Nightly?branchName=master)](https://dev.azure.com/responsibleai/fairlearn/_build/latest?definitionId=23&branchName=master) ![MIT license](https://img.shields.io/badge/License-MIT-blue.svg) ![PyPI](https://img.shields.io/pypi/v/fairlearn?color=blue)
 
-# Fairness in Machine Learning - Mitigation algorithms
 
-A Python package that implements a variety of fairness-related algorithms to mitigate bias including:
+
+Fairlearn
+=============================================================
+Fairlearn is a machine learning fairness toolkit that makes it easier for anyone involved in the development of Artificial Intelligence (AI) systems to monitor and understand the impact of unfairness in their machine learning lifecycle, and use state-of-the-art techniques to mitigate the observed unfairness. Fairlearn repository contains an SDK and Jupyter notebooks with examples to showcase its use.
+
+- [Overview of Fairlearn](#intro)
+- [Target Audience](#target)
+- [Fairness Assessment Dashboard](#getting-started)
+- [Unfairness Mitigation Techniques](#models)
+- [Install Fairlearn](#install)
+- [Contributing](#contributing)
+- [Code of Conduct](#code)
+- [Build Status](#build-status)
+- [Additional References](#refs)
+
+
+
+ 
+# <a name="intro"></a>
+# Overview of Fairlearn
+
+We are at an age where many processes and applications have become or are becoming automated by AI systems. AI is the product of human processes and decisions used to create it, the data used to train it, and the environment used to test it.  The AI system can exhibit different, sometimes harmful, behaviors as a result of this process and systematically discriminate against people based on certain attributes protected by law (e.g., gender, ethnicity). Therefore, regulated industries, or any other industry with legal, moral and ethical responsibilities, need tools to ensure the fairness of models and the predictions they produce. 
```

Comment:
"We are at an age" -> "We are in an age" (it's not _our_ age in years....)
"AI is the product" -> I would not start a sentence with an acronym
In the same sentence, I would emphasise the training data and test environment are also coming from humans, not descending from on high
"The AI system" -> "An AI system" or "Any AI system"
I would not emphasise the attributes being protected by law in that way (dropping an SF reference... compare Daneel's concept of justice in Caves of Steel). The whole thrust of "Future Computed" was about doing the Right Thing, not just meeting legal requirements.